src/ResourceManager/Automation/Commands.Automation/Cmdlet/GetAzureAutomationHybridWorkerGroup.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Security.Permissions;
+using Microsoft.Azure.Commands.Automation.Model;
+using Microsoft.Azure.Commands.Automation.Common;
+
+
+namespace Microsoft.Azure.Commands.Automation.Cmdlet
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRMAutomationHybridWorkerGroup", DefaultParameterSetName = AutomationCmdletParameterSets.ByAll)]
+    [OutputType(typeof(HybridRunbookWorkerGroup))]
+    public class GetAzureAutomationHybridWorkerGroup : AzureAutomationBaseCmdlet
+    {
+        [Parameter(ParameterSetName = AutomationCmdletParameterSets.ByName,Position = 2,  Mandatory = false, ValueFromPipeline = true, HelpMessage = "The Hybrid Runbook Worker Group Name")]
+        [Alias("Group")]
+        public string Name { get; set; }
+
+        protected override void AutomationProcessRecord()
+        {
+            IEnumerable<HybridRunbookWorkerGroup> ret = null;
```

Comment:
you declare ret out here but only use it in one of the if statements. also, result would be better than ret